.github/workflows/merge-gate.yml

```diff
@@ -0,0 +1,46 @@
+name: Merge-Gate
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  fast-checks:
+    name: Fast
+    uses: ./.github/workflows/checks.yml
+
+  slow-checks:
+    name: Slow
+    runs-on: ubuntu-latest
+
+    # Even though the environment "manual-approval" will be created automatically,
+    # it still needs to be configured to require interactive review.
+    # See project settings on GitHub (Settings / Environments / manual-approval).
+    environment: manual-approval
+
+    # Replace the steps below with the required actions 
+    # and/or add additional jobs if required
+    # Note: 
+    #   If you add additional jobs, make sure they are added as a requirement 
+    #   to the approve-merge job's input requirements (needs).
+    steps:
+      - name: Tests
+        run: |
+          echo "Slow tests ran successfully"
```

Comment:
Do we need this whole manual approval mechanism if there are (apparently) no slow tests for this project?

Reply:
By default, without additional configuration, it will pass without requiring approval. This is the standard template, and keeping it there, even though the default does nothing, is good for maintaining workflow synchronization. For those who also want to enable explicit approval, they need to adjust the GitHub project settings and add a reasonable check instead of the echo.